contrib/delta-kernel-rs-cmake/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ else()
     set(profile "release")
 endif()
 
-corrosion_import_crate(MANIFEST_PATH "${DELTA_KERNEL_RS_SOURCE_DIR}/ffi/Cargo.toml" NO_STD PROFILE ${profile} CRATE_TYPES staticlib ALL_FEATURES FLAGS --offline)
+corrosion_import_crate(MANIFEST_PATH "${DELTA_KERNEL_RS_SOURCE_DIR}/ffi/Cargo.toml" NO_STD PROFILE ${profile} CRATE_TYPES staticlib ALL_FEATURES FLAGS --offline --crate-type=staticlib)
```

Comment:
This is what fixes the cross-compilation. Otherwise the dynamic library is built which fails for some obscure reason (most likely related to sysroot). Since we don't care about them as we won't use them, ask to build only the static library

Reply:
https://github.com/ClickHouse/ClickHouse/pull/77604